sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2172,4 +2172,21 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-19809 NullPointerException on zero-size ORC file") {
```

Comment:
This test case should not be put in SQLQuerySuite.scala

Reply:
BTW, @gatorsmile . Which suite do you prefer? So far, this test case covers
- Both `native` and `hive`
- Also, `STORED AS` with `CONVERT_METASTORE_ORC=true`

Reply:
I'll move this into `HiveOrcQuerySuite`.

Reply:
Sure. That sounds fine to me.